Makefile

```diff
@@ -9,6 +9,7 @@ ENABLE_KVM ?= 1
 GDB_TCP_PORT ?= 1234
 INTEL_TDX ?= 0
 RELEASE_MODE ?= 0
+RELEASE_LTO ?= 0
```

Comment:
Could you rename the `RELEASE_LTO` variable (potentially `RELEASE_MODE` as well) to make it more apparent that `RELEASE_LTO` is `RELEASE_MODE` plus LTO? For example, `RELEASE_MODE_WITH_LTO` and `RELEASE_MODE`. Or in a more compact form: `RELEASE` and `RELEASE_LTO`.